perun/cli_groups/import_cli.py

```diff
@@ -106,30 +123,83 @@ def perf_group(ctx: click.Context, **kwargs: Any) -> None:
     help="Runs the conversion of the data in sudo mode.",
     default=False,
 )
-def from_binary(ctx: click.Context, imported: list[str], **kwargs: Any) -> None:
-    """Imports Perun profiles from binary generated by `perf record` command"""
+def from_binary(ctx: click.Context, import_entries: list[str], **kwargs: Any) -> None:
+    """Imports Perun profiles from binary generated by `perf record` command.
+
+    Multiple import entries may be specified; an import entry is either a profile entry
+
+      'profile_path[,<exit code>[,<stat value>]+]'
+
+    where each stat value corresponds to a stats header specified in the --stats-headers option,
+    or a CSV file entry
+
+      'file.csv'
+
+    where the CSV file is in the format
+
+      #Profile,Exit_code[,stat-header1]+
+      profile_path[,<exit code>[,<stat value>]+]
+      ...
+
+    that combines the --stats-headers option and profile entries.
+    """
     kwargs.update(ctx.obj)
-    imports.import_perf_from_record(imported, **kwargs)
+    imports.import_perf_from_record(import_entries, **kwargs)
 
 
 @perf_group.command("script")
-@click.argument("imported", type=str, nargs=-1, required=True)
+@click.argument("import_entries", type=str, nargs=-1, required=True)
 @click.pass_context
-def from_text(ctx: click.Context, imported: list[str], **kwargs: Any) -> None:
-    """Import Perun profiles from output generated by `perf script` command"""
+def from_text(ctx: click.Context, import_entries: list[str], **kwargs: Any) -> None:
+    """Import Perun profiles from output generated by `perf script` command.
+
+    Multiple import entries may be specified; an import entry is either a profile entry
+
+      'profile_path[,<exit code>[,<stat value>]+]'
+
+    where each stat value corresponds to a stats header specified in the --stats-headers option,
+    or a CSV file entry
+
+      'file.csv'
+
+    where the CSV file is in the format
+
+      #Profile,Exit_code[,stat-header1]+
+      profile_path[,<exit code>[,<stat value>]+]
+      ...
+
+    that combines the --stats-headers option and profile entries.
+    """
     kwargs.update(ctx.obj)
-    imports.import_perf_from_script(imported, **kwargs)
+    imports.import_perf_from_script(import_entries, **kwargs)
 
 
 @perf_group.command("stack")
-@click.argument("imported", type=str, nargs=-1, required=True)
+@click.argument("import_entries", type=str, nargs=-1, required=True)
 @click.pass_context
-def from_stacks(ctx: click.Context, imported: list[str], **kwargs: Any) -> None:
+def from_stacks(ctx: click.Context, import_entries: list[str], **kwargs: Any) -> None:
     """Import Perun profiles from output generated by `perf script | stackcollapse-perf.pl`
-    command
+    command.
+
+    Multiple import entries may be specified; an import entry is either a profile entry
+
+      'profile_path[,<exit code>[,<stat value>]+]'
```

Comment:
The parentheses represent optional parts of the import entries. This change was done so that (a) CSV and CLI specification of profiles to import is unified, i.e., it is now possible to specify exactly the same import parameters using both CSV files and CLI, and (b) at least exit code may be specified on CLI alongside the profiles to import. However, the interface is backwards-compatible and simple profile paths, e.g., `import.stack.gz` may still be specified. An example input containing both exit code and some stats values (corresponding to some specified stats headers defined in option `--stats-headers`) is, e.g., `'import.stack.gz,0,18511.379883,367'`.